protocol-units/execution/maptos/opt-executor/src/background/transaction_pipe.rs

```diff
@@ -71,6 +75,14 @@ impl TransactionPipe {
 				mempool_config.sequence_number_ttl_ms,
 				mempool_config.gc_slot_duration_ms,
 			),
+			whitelisted_accounts: mempool_config.whitelisted_accounts()?,
```

Comment:
Why does this need to be fallible? Wouldn't it be better to have Config correct on construction and `whitelisted_accounts` a simple getter?